app/scripts/background.js

```diff
@@ -294,16 +294,20 @@ function maybeDetectPhishing(theController) {
         blockReason = blockedRequestResponse.type;
       }
 
+      const blockedUrl =
+        blockReason === 'c2DomainBlocklist' ? details.initiator : hostname;
+
```

Comment:
So there's a couple of things here:
1. We should try to avoid having inline strings spread out, as it will make it much harder to update the blockReason value in the future (we would need to update the controller, and then remember to search for the actual string value in the Extension codebase). Here we could use the type from the PhishingController (if it is not exported, we can export it).
2. we already have an if block above where we set a specific blockReason. And here we are adding another if clause for the blockReason to set the blockedUrl. We should collapse into the pre-existing if block that sets the blockReason and set the blockUrl as well.

Suggestion, update the above if block as below:
```js
      let blockReason;
      let blockedUrl = hostname;
      if (phishingTestResponse?.result && blockedRequestResponse.result) {
        blockReason = `${phishingTestResponse.type} and ${blockedRequestResponse.type}`;
      } else if (phishingTestResponse?.result) {
        blockReason = phishingTestResponse.type;
      } else {
        blockReason = blockedRequestResponse.type;
        blockedUrl = details.initiator;
      }
```

Let me know your thoughts?

Reply:
Love this, definitely makes the logic much cleaner. Thanks - will update this